frontend/src/theme/components/DropdownMenu.ts

```diff
@@ -0,0 +1,76 @@
+import { ComponentMultiStyleConfig } from '@chakra-ui/react'
+
+export type DropdownMenuVariant = 'outline' | 'clear'
+export type DropdownMenuSize = 'normal' | 'stretch'
+
+export const DropdownMenu: ComponentMultiStyleConfig = {
+  parts: ['outerbox', 'innerbox', 'option'],
+  baseStyle: ({ isActive }) => ({
+    outerBox: {
+      padding: '4px',
+      background: 'white',
+      borderRadius: '8px',
+      border: '0px',
+      width: '177px',
+      color: 'secondary.500',
+      _hover: {
+        background: 'white',
+      },
+      _focus: {
+        background: 'secondary.300',
+      },
+      _active: {
+        background: 'white',
+      },
+    },
+    innerBox: {
+      display: 'flex',
+      flexDirection: 'row',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+      padding: '8px 16px',
+      height: '44px',
+      background: 'white',
+      boxSizing: 'border-box',
+      borderStyle: 'solid',
+      borderWidth: isActive ? '2px' : '1px',
+      borderRadius: '4px',
+      borderColor: 'white',
+    },
+    option: {
+      _hover: {
+        background: 'primary.100',
+      },
+      _focus: {
+        background: 'white',
+        borderWidth: '2px',
+        borderStyle: 'solid',
+        borderColor: 'primary.500',
+      },
```

Comment:
i'm not entirely sure this is correct for 2 reasons.

1. setting `borderWidth` here will cause the element to resize. this leads to it shrinking.
2. next, setting background color doesn't seem correct because css pseudo classes are allowed to overlap (focus/active being the most obvious ones). 

i think the focus styling is actually only the border, and a trick to work around the resizing might be to use `boxShadow`. tbh, i'm not entirely sure that's the best way also. because the item doesn't have an actual border, another way could be to have a 2px transparent border and only show colour on focus (will avoid resizing)